src/app/pages/properties/properties.component.ts

```diff
@@ -0,0 +1,47 @@
+import { PropertiesService } from './../../shared/services/properties/properties.service';
+import { Component, OnInit } from "@angular/core";
+import { Property } from "../../models/Property";
+
+@Component({
+  selector: "app-properties",
+  templateUrl: "./properties.component.html",
+  styleUrls: ["./properties.component.scss"]
+})
+export class PropertiesComponent implements OnInit {
+  properties: Property[];
+  propertiesUrl = "http://127.0.0.1:8000/api/v1/properties/";
```

Comment:
Hello @Genza999, could you use the live Heroku url such that it's easier to test the functionality on the review app also.

Reply:
Thanks @CryceTruly, I have worked on this